src/routes/landing/components/LocationEntry.js

```diff
@@ -1,9 +1,8 @@
 import PropTypes from 'prop-types'
```

Comment:
if you want you could rename this to CityEntry (also LocationSelector -> CitySelector, LocationListItem -> CityListItem and so on)